src/view/render.rs

```diff
@@ -96,69 +96,60 @@ fn render_message(ctx: &mut Context, app: &mut ApplicationState) -> GameResult<(
     if let Some(ref message) = app.game_state.message {
         let Message { title, subtitle } = *message;
         let Size { width, height } = app.game_state.world.size;
-
-        let w = width / 2.0;
-        let h = height / 2.0;
-
-        let mut draw_text = |text: &str, color: graphics::Color, is_title: bool| {
-            let drawable = graphics::Text::new(ctx, text, &app.resources.font).unwrap();
-            let width = w - (drawable.width() as f32 / 2.0);
-            let height = if is_title {
-                h - drawable.height() as f32
-            } else {
-                h
+        let center_y = height / 2.0;
+
+        let mut draw_text =
+            |text: &str, color: graphics::Color, is_title: bool| -> GameResult<()> {
+                let mut text = graphics::Text::new(graphics::TextFragment {
+                    text: text.to_owned(),
+                    font: Some(app.resources.font),
+                    scale: Some(graphics::Scale::uniform(28.0)),
+                    ..Default::default()
+                });
+
+                let x = 196.0;
```

Comment:
I'm really not sure why I had to pad this text by `196px`... Unless I'm doing something wrong with the calculations? It's either that or there's a bug in `ggez` with aligning text (or I've misunderstood how `text.set_bounds()` is meant to work).

Reply:
No idea either. I would suggest asking on discord or other place. Maybe we are missing something. Otherwise we can just leave it as is.